projects/canopy/src/styles/typography.notes.ts

```diff
@@ -2,12 +2,43 @@ export const notes = `
 # Typography
 
 ## Purpose
-Provides common typographic styles for native elements with additional utility classes
+Provides common typographic styles for native elements with additional utility classes.
+
+Canopy consists of two fonts:
+
+* **Roboto**: Productive font, for general use in self serve apps (default)
+* **Lyon**: Expressive font, for headings in promo material (applied by class modifier)
+
+> **IMPORTANT** - Lyon font files are not included in the Canopy package, you have to serve it yourself, see below for details.
+
+## Serving the Lyon font files
+
+Due to our commericial license agreement, Lyon is not included in the Canopy distributed package, so you need to include it in the **assets** directory of your application. Canopy still references Lyon in the \`\`@font-face\`\` rulesets, so you don't need to implement them, but you do need to ensure that the path to the font in your assets directory is the same as the paths specified by Canopy in \`\`@font-face\`\`.
```

Comment:
The repetition of documentation between storybook and github is a bit annoying, but maybe one for a later date. Ideally all of our docs will be in one place (even if they are visible in Github and Storybook)

Reply:
Yes good point. I wonder if we could have snippets of markdown that can be imported and used in more than one place if needed. Shall we capture this in an issue?

Reply:
Could be worth tracking, a hard one to fix, but it might link partially to #35